api/src/pcapi/core/external_bookings/boost/client.py

```diff
@@ -1,3 +1,5 @@
+from datetime import date
+from datetime import timedelta
```

Comment:
```suggestion
import datetime
```
Cela allonge un peu les lignes, mais `black` est là pour nous.